handlebarsHelpers.js

```diff
@@ -43,5 +43,15 @@ module.exports = {
     });
 
     return Object.values(speakers);
+  },
+  'getSponsors': function(events) {
+    const sponsors = {}
```

Comment:
`Minor` Missing semicolon; 'cos, you know, semicolons are super important :laughing:

Reply:
oh the shame! 😞